src/mpcCoreKit.ts

```diff
@@ -318,16 +318,19 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     if (this.isNodejsOrRN(this.options.uxMode)) {
       throw CoreKitError.oauthLoginUnsupported(`Oauth login is NOT supported in ${this.options.uxMode} mode.`);
     }
-    const { importTssKey } = params;
+    const { importTssKey, registerExistingSFAKey } = params;
     const tkeyServiceProvider = this.torusSp;
-
+    if (this.isRedirectMode && (importTssKey || registerExistingSFAKey)) {
+      throw CoreKitError.invalidConfig("key import is not supported in redirect mode");
+    }
```

Comment:
check for both importTssKey and registerExistingSFAKey for loginwithOAuth
It is check in loginwithJwt

```

    if (registerExistingSFAKey && importTssKey) {
      throw CoreKitError.invalidConfig("Cannot import TSS key and register SFA key at the same time.");
    }
    ```

Reply:
done